src/Mod/Path/PathScripts/PathCircularHoleBase.py

```diff
@@ -650,7 +603,8 @@ def visualAxis(self):
         if not FreeCAD.ActiveDocument.getObject('xAxCyl'):
             xAx = 'xAxCyl'
             yAx = 'yAxCyl'
-            FreeCAD.ActiveDocument.addObject("App::DocumentObjectGroup", "visualAxis")
+            # zAx = 'zAxCyl'
+            VA = FreeCAD.ActiveDocument.addObject("App::DocumentObjectGroup", "visualAxis")
```

Comment:
variables should not start with an upper case, unless they are public and global.
All caps is generally thought off as a constant.

Reply:
Fixed.